server/routes/api/browser.py

```diff
@@ -110,3 +110,52 @@ def get_observation_ids():
         dcid = cells[0].get('value', '')
         result[obsDate] = dcid
     return Response(json.dumps(result), 200, mimetype='application/json')
+
+
+def statvar_hierarchy_helper(svg_id, sv_groups, processed_svg, processed_sv,
+                             seen_nodes):
+    sv_group = sv_groups.get(svg_id, {})
+    for child_sv in sv_group.get("childStatVars", []):
+        if child_sv in seen_nodes:
+            continue
+        child_sv_node = {}
```

Comment:
child_sv_node -> sv2svg; "node" is too general here and does not tell what it does :)

Reply:
done